extensions/standard-processors/processors/PutTCP.h

```diff
@@ -71,14 +71,11 @@ class PutTCP final : public core::ProcessorImpl {
       .withValidator(core::StandardPropertyValidators::TIME_PERIOD_VALIDATOR)
       .withDefaultValue("15 seconds")
       .isRequired(true)
-      .supportsExpressionLanguage(true)
       .build();
   EXTENSIONAPI static constexpr auto Timeout = core::PropertyDefinitionBuilder<>::createProperty("Timeout")
       .withDescription("The timeout for connecting to and communicating with the destination.")
       .withValidator(core::StandardPropertyValidators::TIME_PERIOD_VALIDATOR)
-      .withDefaultValue("15 seconds")
```

Comment:
Why is this default value removed?  If we want to remove it, it should be removed from PROCESSORS.md, too.